### ISSUE TITLE
server, storage, ui: add the lease state to the range debug page

### ISSUE THREAD

**Issue by BramGruneir** [cockroachdb/cockroach#18339]
This would be very helpful.  No Lease is a good summary, but it only means that the lease state is not leaseValid.

**Comment by tschottdorf** [comment]
Example page: this reports an error, but everything looks fine (?)

![image](https://user-images.githubusercontent.com/5076964/30241110-25f4445a-954b-11e7-9a06-ad3623a07da1.png)

**Comment by tschottdorf** [comment]
@dianasaur323 should really try this for 1.1. Not a lot of work, just a little bit of plumbing. Seeing this in #18324 and want to make sure it's not a real issue.

**Comment by BramGruneir** [comment]
Tobi is right, should be an easy addition and very helpful. Just a bit of plumbing.  I want to also add in if any replicas are not considered alive, which will give insight as to why the state may be what it is.  Since this will be plumbing in the same area.

But both of these should wait until after #18376 is merged, since they're touching some of the same parts.  That's just waiting for a review still.

**Comment by dianasaur323** [comment]
Another issue occurs when a range hasn't been servicing any traffic, and it was both a raft leader and leaseholder. When something changes, there is a situation where another node become the raft leader, but the leaseholder is not moved because it wasn't servicing any traffic. However, directly running query to that range moves the leaseholder over. We want to minimize false positives, so we should probably address this issue.

**Comment by petermattis** [comment]
We could unquiesce a range when leader-not-leaseholder is detected. When the range quiesces it will try to move the leader to the leaseholder. Seems straightforward to do this, though I wonder if there will be unforeseen side-effects.
